components/authz-service/server/v2/project.go

```diff
@@ -70,7 +77,15 @@ func NewPostgresProjectsServer(
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to initialize v2 store state")
 	}
-	projectUpdateManager := NewProjectUpdateManager(eventServiceClient, configManager)
+	//projectUpdateManager := NewProjectUpdateManager(eventServiceClient, configManager)
+	projectUpdateManager, err := RegisterCerealProjectUpdateManager(projectUpdateCerealManager,
+		"ProjectUpdate", "Runner", []string{
```

Comment:
"Runner" is the `instanceName` variable for the workflow, should that be a unique ID for each Automate node?

Reply:
I don't think I know what the `instanceName` does.

Reply:
unique id would be problematic. By using a constant instance name, we guarantee only one of these runs at a time

Reply:
singleton is probably a better name